packages/terra-worklist-data-grid/tests/jest/WorklistDataGrid.test.jsx

```diff
@@ -1,35 +1,58 @@
 // eslint-disable no-console
 import React from 'react';
 /* eslint-disable-next-line import/no-extraneous-dependencies */
+import { shallowWithIntl } from 'terra-enzyme-intl';
 import WorklistDataGrid from '../../src/WorklistDataGrid';
 import dataFile from './testData.json';
 
 describe('WorklistDataGrid', () => {
-  it('renders a grid with 3 columns and 3 data rows', () => {
-    const wrapper = shallow(
+  it('renders a grid with 3 columns and 3 data rows and 9 cells', () => {
+    const wrapper = shallowWithIntl(
       <WorklistDataGrid
         id="test-terra-worklist-data-grid"
         columns={dataFile.cols}
         rows={dataFile.rows}
       />,
-    );
+    ).dive();
 
     const columnHeaders = wrapper.find('.worklist-data-grid-column-header');
-    const rowHeaders = wrapper.find('.worklist-data-grid-row-header');
+    const rowHeaderCells = wrapper.find('th.worklist-data-grid-cell');
+    const dataGridCells = wrapper.find('.worklist-data-grid-cell');
+
+    // Test masked cells
+    const maskedCells = wrapper.find('.masked');
 
     expect(columnHeaders).toHaveLength(3);
-    expect(rowHeaders).toHaveLength(3);
+    expect(rowHeaderCells).toHaveLength(3);
+    expect(dataGridCells).toHaveLength(9);
+    expect(maskedCells).toHaveLength(3);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('test masked cells', () => {
```

Comment:
Test descriptions should read as a sentence beginning with "it".
```suggestion
  it('renders a grid with masked cells', () => {
```